### PR TITLE
fix(no-useless-v-bind): remove spaces around string in fixes

### LINKED ISSUE #2396: `vue/no-useless-v-bind` auto-fix adds extra whitespace
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 8.56.0
- **eslint-plugin-vue version:** 9.21.1
- **Vue version:** 
- **Node version:** 
- **Operating System:** 

**Please show your full configuration:**

```json5
module.exports = {
    extends: ['plugin:vue/recommended'],
    rules: {
        'vue/no-useless-v-bind': 'error',
    },
};
```

**What did you do?**	

```vue
<template>
  <div 
    :title="
      'text'
    "
  />
</template>
```

**What did you expect to happen?**

```vue
<template>
  <div 
    title="text"
  />
</template>
```

**What actually happened?**

```vue
<template>
  <div 
    title="
      text
    "
  />
</template>
```

**Repository to reproduce this issue**

[Playground](https://ota-meshi.github.io/eslint-plugin-vue-demo/#eJx1lMFynDAMhl9lh0su8WSmx0zaJ8lFmB9QYmwq27tLM3n3Csga2KanXX2yhfRL8kdlQ4PquXpJGEZHCb9e/en00vD5NP85nZ4TJ4efr9Vqnk4PCdf0sForfdI7L08lQPVYSXaI1fNHdc54smEY4JNpWGATn/VzPx4Xz1u8mhwRzZkkLtQGH5Nkm4KYmEfIQlu1vm4Hv5AOKUGMIGVZiQ+G4uStGSUMHGFwhc0a5+a1pLCBdSQ0x4nF4ShGvRy5K6E055EExqMzfyBh477556im/B0jrXk+zyXr1ZMkOM28w/UGG9S56yCb7ZAwq1KIamFIungAa+YDhhpy9MBpsdwe2Dum/SHHVru1yHKj2sE0HQxje5XLzlovHzs6R5q7UGrD9U4GHRQhU4fgQH7+Urp5WnIu9RJy1xeUtXnH+50LNbk7yMMY5F5w9l5T/K677M/kuFkFHwsVtbMeNpeeE+JItqjggooaWh0kWI673ulYaSEN++5/snhcTJyGOrhCgg82aRGaGw/6i2hpLN8K9Zvq4lyJsBy+GTrKKaRJe1dndom3qgRrrdiAlmeWOsqhCNfe6RS/25vYUxMuCnX1WItqjKdhF0Y3Ic7TJ7QNUOo5mhq6mdgtqjqyb9BuhmqOJeSQtQDHHptPQKpj7XYoUgvTsldJpjuqq0j7TfrCYZyhCquh2GtvNr8+LY1xVGOT9wuW92ZGKhO04zXZd0ELgd9m4ebUVbH9PTy28sJpPSH4nfWxMhPDNQvJ8zJGT2vu6zzObQ1teQp3Yc+mZj9f/Pz8C+Aj6Ao=)

### PR DESCRIPTION
**Pull request by claneo**
Fixes #2396

### CODE REVIEW

File: tests/lib/rules/no-useless-v-bind.js [hunk 1]
```diff
@@ -136,6 +136,22 @@ tester.run('no-useless-v-bind', rule, {
         'Unexpected `v-bind` with a string literal value.',
         'Unexpected `v-bind` with a string literal value.'
       ]
+    },
+    {
+      code: `
+      <template>
+        <div :id="  'foo'  " />
+        <div :id='  "foo"  ' />
```
Comment (FloEdelmann) [inline tests/lib/rules/no-useless-v-bind.js:144]: Can you please add these test cases here? Are they handled correctly?

```pug
<div :id=" \"foo\" " />
<div :id="' \'foo\' " />
<div :id=" `foo` " />
<div :id=' `foo` ' />
```

Comment (claneo) [reply tests/lib/rules/no-useless-v-bind.js]: All of these test cases were fixed correctly, except the first one that seems invalid.

Comment (FloEdelmann) [reply tests/lib/rules/no-useless-v-bind.js]: Oh yeah, that was a typo on my side. I removed those test cases again, as this rule should not care about invalid v-bind values; there is [`vue/valid-v-bind`](https://eslint.vuejs.org/rules/valid-v-bind.html) for that.

Thanks for checking!

**Review by FloEdelmann — APPROVED** [review]

**Review by ota-meshi — APPROVED** [review]
LGTM! Thank you.

**Pull request merged**